lib/matplotlib/tight_layout.py

```diff
@@ -90,8 +90,8 @@ def auto_adjust_subplotpars(
         else:
             margin_top = None
 
-    vspaces = [[] for i in range((rows + 1) * cols)]
-    hspaces = [[] for i in range(rows * (cols + 1))]
+    vspaces = [[] for _ in range((rows + 1) * cols)]
+    hspaces = [[] for _ in range(rows * (cols + 1))]
```

Comment:
Initially, I tried to shorten these further to
~~~
vspaces = [[]] * ((rows + 1) * cols)
hspaces = [[]] * (rows * (cols + 1))
~~~
But for some reason, this caused test failures.

Reply:
Unfortunately, it is not the same thing:
```
>python -c "a = [[]] * 2; a[0].append(123); print(a)"
[[123], [123]]
```

P.S: I think those can be turned into numpy arrays of python lists.

Reply:
Thanks, of course it's not the same. :roll_eyes: 

I leave numpy-fication for later. I suspect one could use numpy also further down to speed things up a little, but it needs a closer look.